install.js

```diff
@@ -145,3 +145,12 @@ function generateProtocolTypesIfNecessary(updated) {
     return;
   return require('./utils/protocol-types-generator');
 }
+
+function log(toBeLogged) {
```

Comment:
nit: I'd rename log to logPolitely or to be a little easier to read what this does.

Reply:
I like logPolitely, I’ll use that.